src/main/resources/db/migration/all/20230525131850__add_risk_fields_to_cas_2_applications.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE cas_2_applications ADD COLUMN risk_ratings JSON;
```

Comment:
Would just include this in the CREATE TABLE DDL in `20230525120902__cas_2_applications.sql`